tools/lsp/ui/views/property-view.slint

```diff
@@ -428,14 +428,13 @@ component ExpandableGroup {
                         }
 
                         if property.value.kind == PropertyValueKind.code: VerticalLayout {
-                            padding-top: 2px;
 
                             NameLabel {
                                 property-information: property;
                                 element-information: root.element-information;
                             }
 
-                            spacing: EditorSpaceSettings.default-spacing;
+                            padding-bottom: EditorSpaceSettings.default-spacing;
```

Comment:
Ideally this would not be mixed in between Elements but be at the top of the block with all other properties.

Just a nit-pick :-)